docs/source/configuration.rst

```diff
@@ -350,6 +350,74 @@ policy.
 
         *"It's some arcane magic from AT LEAST a decade ago."*
 
+Loop prevention
+---------------
+
+In order to prevent the bot from entering a loop (for example when there is
+another bot in the same channel, or if a user spams a command), it'll try to
+see if the next message to send is repeating too often in the last eight
+messages in the last few minutes. If that happens, the bot will send ``...``
```

Comment:
Here's a thought: Why both a time _and_ message-count threshold? With a configurable `antiloop_window` and `antiloop_threshold`, I can't think of any reason to hard-code looking at only the last 8 messages.